deploy/gk-deploy

```diff
@@ -233,6 +233,7 @@ while [[ $# -ge 1 ]]; do
         ;;
         -abort)
         ABORT=1
+        SKIP_PREREQ=1
```

Comment:
@MohamedAshiqrh the SKIP_PREREQ should be used for the same purpose even later. Also I feel the flag has to be enabled as soon as the option is provided. but agree on, yes there are different ways to fix it.